doc/guide/static/manifest.schema.json

```diff
@@ -0,0 +1,256 @@
+      "type": "string"
+    },
+    "name": {
+      "description": "An optional string that changes the name of the package. Normally packages derive their name from the directory that they are located in. This field overrides that name.",
+      "type": "string"
+    },
+    "priority": {
+      "type": "number",
+      "description": "An optional number that specifies which package is preferred in cases where there are conflicts. For example given two packages with the same `name` a package is chosen based on its priority."
+    },
+    "conditions": {
+      "description": "An optional list of `{\"predicate\": \"value\"}` objects. Cockpit will only consider the package if all conditions are met. Currently supported predicates are `path-exists` and `path-not-exists`. Unknown predicates are ignored. This is preferable to using `priority`, but only available since Cockpit 286.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "path-exists": {
+            "description": "A condition where a specific file path must exist.",
+            "type": "string"
+          },
+          "path-not-exists": {
+            "description": "A condition where a specific file path must exist.",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "requires": {
+      "type": "object",
+      "description": "An optional JSON object that contains a \"cockpit\" string version number. The package will only be usable if the Cockpit bridge and javascript base are equal or newer than the given version number.",
+      "properties": {
+        "cockpit": {
+          "type": "string",
+          "pattern": "^\\d+$"
+        }
+      },
+      "additionalProperties": {
+        "type": "string"
+      },
+      "required": [
+        "cockpit"
+      ]
+    },
+    "version": {
+      "type": "string",
+      "description": "An informational version number for the package."
+    },
+    "preload": {
+      "type": "array",
+      "description": "A list of identifiers of the components that should be preloaded. Normally, the files of a component are loaded when the user navigates to it for the first time. The files of a preloaded component are loaded immediately after the user logs in, and the initialization code of the component is invoked. \n\nThe value of this field is an array of strings, where each string is one of the keys used in the `dashboard`, `menu`, or `tool` fields."
+    },
+    "parent": {
+      "type": "object",
+      "description": "Used when module does not have its own menu item but is a part of a different module.",
+      "properties": {
+        "component": {
+          "type": "string",
+          "description": "The name of the superordinate component."
+        },
+        "docs": {
+          "$ref": "#/$defs/docs"
+        }
+      }
+    },
+    "tools": {
+      "description": "todo",
```

Comment:
"tools" and "dashboard" should be the same as "menu", i.e."#/$defs/menu".

Reply:
Ack thanks! Lots of lines so explains why I missed a few things